src/Application/Cli/DeployCommand.php

```diff
@@ -93,9 +100,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $sourceDirectory = $input->getArgument('source');
         $repositoryUrl = $input->getOption('repository');
         $targetBranch = $input->getOption('branch');
+        $tempConfigRaw = $input->getOption('config');
 
         $project = new Project($sourceDirectory, getcwd().'/.couscous/generated');
 
+        $project->metadata['tempConfigRaw'] = $tempConfigRaw;
```

Comment:
`tempConfigRaw` doesn't sound ideal to me, how about something like `cliConfig`?